code/game/objects/items/food/bread.dm

```diff
@@ -1,5 +1,7 @@
 
-/obj/item/food/bread
+/obj/item/food/bread //If you'll try to spawn it will occure in error sprite. Shouldn't do this.
```

Comment:
```suggestion
/// Abstract parent object for bread items. Should not be made obtainable in game.
/obj/item/food/bread
```
Just a slight change for grammar/clarity.

Reply:
👍